semantic_conventions.md

```diff
@@ -31,8 +31,9 @@ Span tags apply to **the entire Span**; as such, they apply to the entire timera
 | `peer.ipv4` | string | Remote IPv4 address as a `.`-separated tuple. E.g., `"127.0.0.1"` |
 | `peer.ipv6` | string | Remote IPv6 address as a string of colon-separated 4-char hex tuples. E.g., `"2001:0db8:85a3:0000:0000:8a2e:0370:7334"` |
 | `peer.port` | integer | Remote port. E.g., `80` |
-| `peer.service` | string | Remote service name (for some unspecified definition of `"service"`). E.g., `"elasticsearch"`, `"a_custom_microservice"`, `"memcache"` |
+| `peer.service` | string | Remote service name (for some unspecified definition of `"service"`) being called by a client. E.g., `"elasticsearch"`, `"a_custom_microservice"`, `"memcache"`. Meaning should correspond with values set in `service`. |
```

Comment:
"being called by a client" is incorrect, peer.service tag is symmetric, used by both client and server spans to identify the remote service.

Reply:
That's fair in theory, but I don't understand how that would be useful in practice.  Do people write instrumentation where the server creates a span on behalf of a client like is common for the reverse?

Should I remove the `being called by a client` part then?

Reply:
@yurishkuro done.